### DISCUSSION TITLE
Nightly 2.4 mac version not working

### DISCUSSION THREAD

**Discussion by ocochodzi111** [supermerill/SuperSlicer#2443] [Q&A]
All 2.4 nightly builds not working (mac prebuild versions), they crash right after start, before splash screen. Am i missing something? Mac mini m1.

**Comment by liftbag** [comment]
In my case, running the app from terminal I had this log:

liftmac-3:MacOS paolo$ ./SuperSlicer --loglevel 5
dyld[1405]: Library not loaded: /usr/local/opt/zstd/lib/libzstd.1.dylib
  Referenced from: /Volumes/Macintosh HD - Dati/Dati/paolo/Desktop/SuperSlicer.app/Contents/MacOS/SuperSlicer
  Reason: tried: '/usr/local/opt/zstd/lib/libzstd.1.dylib' (no such file), '/usr/local/lib/libzstd.1.dylib' (no such file), '/usr/lib/libzstd.1.dylib' (no such file)
Abort trap: 6

After that, I brew install zstd and the 2.4 nightly is working. However it corrupts any overwritten profiles, which is why 2.3 can no longer be opened until you manually delete the corrupted profiles.

**Reply by ocochodzi111** [reply]
So it looks like we need little patience here.

**Reply by MarcoZ76** [reply]
> In my case, running the app from terminal I had this log:
> 
> liftmac-3:MacOS paolo$ ./SuperSlicer --loglevel 5 dyld[1405]: Library not loaded: /usr/local/opt/zstd/lib/libzstd.1.dylib Referenced from: /Volumes/Macintosh HD - Dati/Dati/paolo/Desktop/SuperSlicer.app/Contents/MacOS/SuperSlicer Reason: tried: '/usr/local/opt/zstd/lib/libzstd.1.dylib' (no such file), '/usr/local/lib/libzstd.1.dylib' (no such file), '/usr/lib/libzstd.1.dylib' (no such file) Abort trap: 6
> 
> After that, I brew install zstd and the 2.4 nightly is working. However it corrupts any overwritten profiles, which is why 2.3 can no longer be opened until you manually delete the corrupted profiles.

Hi liftbag, i have the same your issue, can you explain me how to install zstd on my mac? Thx in advance

**Reply by liftbag** [reply]
> Hi liftbag, i have the same your issue, can you explain me how to install zstd on my mac? Thx in advance

Have you tried with the nightly fix start crash macos 502?
If still don't work, first, install homebrew. Follow the instruction here:  https://brew.sh/index

Maybe you need to install the command line tools typing in the terminal:
`xcode-select --install`

When installation is complete, in the terminal type:
`brew update`
`brew upgrade`
`brew install zstd`

Important note: as soon as you save a profile with the v2.4, the v2.3 will no longer open until you delete the profile saved with the v2.4. Here the error message:
<img width="794" alt="Schermata 2022-03-09 alle 11 21 52" src="https://user-images.githubusercontent.com/14921654/157422523-1a993c4c-fe65-43bd-a089-3b2da2152370.png">
